tests/test_validators.py

```diff
@@ -115,3 +115,11 @@ def test_unsorted(self):
         status.step(first)
         status.step(second)
         self.assertFalse(sort_validator(status))
+
+    def test_msgctxt(self):
+        first = POEntry(msgid="First", msgstr="Translation")
+        second = POEntry(msgid="First", msgstr="Trans.", msgctxt="abbrev.")
+        status = Status()
+        status.step(first)
+        status.step(second)
+        self.assertTrue(sort_validator(status))
```

Comment:
A test for sorting the messages with the same ID and with and without context should be added to prevent the problem of comparison between `None` and `str`.

Reply:
Test added.